examples/main.tf

```diff
@@ -52,11 +52,18 @@ module "functionbeat" {
       security_group_ids = [aws_security_group.functionbeat_securitygroup.id]
     }
 
+    # choose either elasticsearch output
     output_elasticsearch = {
-      hosts : ["https://your-endpoint:443"]
-      protocol : "https"
-      username : "elastic"
-      password : "mysupersecret"
+      hosts    = ["https://your-endpoint:443"]
+      protocol = "https"
+      username = "elastic"
+      password = "mysupersecret"
+    }
+
+    # or choose logstash output.
+    output_logstash = {
```

Comment:
since multiple outputs are not supported, i suggest we should provide a dedicated example for the logstash_output use case, to ensure the examples are actually deployable.

Reply:
Good idea, I can do that. How would you like it? 

* A separate `resource` block with only output_logstash in `example/main.tf` or 
* Two separate files `example/elasticsearch.tf` and `example/logstash.tf` 

?

Reply:
Let's have separate files for the examples 👍